icicle/appUtils/msm/msm.cu

```diff
@@ -530,7 +542,7 @@ namespace msm {
       CHK_IF_RETURN(cudaFreeAsync(sort_single_temp_storage, stream));
 
       // find large buckets
-      unsigned avarage_size = size / (1 << c);
+      unsigned avarage_size = msm_size / (1 << c);
```

Comment:
making sure that this is the size of a single msm

Reply:
yes it is. I will rename variables to make the code clearer.